README.md

```diff
@@ -78,8 +78,8 @@ Additional annotation information and external identifiers for Human-GEM reactio
 * Load and save using the built-in MATLAB `load()` and `save()` functions.
 
 `Human-GEM.yml` (Recommended if on `devel` or other branches)
```

Comment:
Shouldn't this be `develop`?

Reply:
nice catch, this is fixed in 0c2f0baf972940984d3ca67092dc14feade63b34